interface.py

```diff
@@ -0,0 +1,41 @@
+import pygame
+import serial
+import sympy
+import numpy
+import matplotlib
+import PySimpleGUI as sg
+import time
+from threading import Thread
+
+class Interface():
+    
+    def __init__(self, robot, object_seen, motor_controls):
+        
+        self.robot = None
```

Comment:
Should this be `self.robot = robot`, etc.?

Reply:
Yeah